kolibri/plugins/learn/assets/src/views/TopicsPage.vue

```diff
@@ -371,6 +346,8 @@
   import { normalizeContentNode } from '../modules/coreLearn/utils.js';
   import useSearch from '../composables/useSearch';
   import genContentLink from '../utils/genContentLink';
+  import HeaderTabs from '../../../../coach/assets/src/views/common/HeaderTabs';
+  import HeaderTab from '../../../../coach/assets/src/views/common/HeaderTabs/HeaderTab';
```

Comment:
Glad this was a straight-forward drop-in replacement!

Preferably we'd move this to core API spec (to avoid cross-plugin references) or even KDS (to make this component fully documented)

I'll let you and @rtibbles decide whether that should happen now or later

Reply:
I'm going to vote for later. Thanks to our updated string machinery, cross plugin imports like this are less of a concern, but the long import paths are not great.

Also good to think about the contrast between code used in common between apps vs things we want to expose as a common API.

I think most of our core API spec is the former at the moment, whereas it _should_ be exclusively the former.